BEACON-V2/genomicVariations/examples/genomicVariant-MIN-VRS-example.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "../defaultSchema.json",
+  "variantInternalId": "GRCh37-1-55505652-G-A",
+  "variation":{
+    "state": {
+      "sequence": "A",
+      "type": "SequenceState"
```

Comment:
This is deprecated in VRS, `LiteralSequenceExpression` should be used instead from what I can tell.

Reply:
As I understand the schema this is correct here: `state` is a `SequenceExpression` (or alternatively a `SequenceState`) which can be a `LiteralSequenceExpression` which has the properties `type` and `sequence`. So`state` collapses to those 2 properties.

Reply:
That's what I thought at first as well but then in the [vrs.yaml file](https://github.com/ga4gh/vrs/blob/main/schema/vrs.yaml) SequenceState is set to deprecated: true (line 619-20).